packages/calypso-e2e/src/jest-playwright-config/setup.ts

```diff
@@ -1,5 +1,9 @@
 import { jest } from '@jest/globals';
 import config from 'config';
 
-// Set the default timeout value
-jest.setTimeout( config.get( 'jestTimeoutMS' ) );
+if ( process.env.PWDEBUG ) {
```

Comment:
Tiny nit, but I believe we might want to check for values explicitly here. All ENV vars are strings, and "0" is technically truthy, so `PWDEBUG=0` would actually trigger this line!

Reply:
Good catch, thanks!